### DISCUSSION TITLE
Hide the default calendar module

### DISCUSSION THREAD

**Discussion by ZanettaPM** [MMRIZE/MMM-CalendarExt3#104] [General]
Hi all, the module is working great for me. Thanks to the dev! However, because it requires the default calendar module, I am kind of stuck with the old calendar showing up in the top_right corner of the screen. 

Is there a way to hide the default calendar ? 

I tried reducing the number of events to forecast but it impacts the ext3 calendar as well if I change this value in the calendar module section. 

Thanks a lot

**Comment by eouia** [comment]
Remove position from default calendar module.

**Reply by ZanettaPM** [reply]
Such an easy solution ... ahah ! Thanks a lot !

**Reply by ZanettaPM** [reply]
Maybe you can let this discussion open as it can be a question that come back again ? I did not mention that another discussion with the same topic existed as it was marked as "closed". 

Anyway your choice, 

Best,

**Reply by eouia** [reply]
It would be better to have one thread for one issue.
Of course you can reopen the same issue. Or you can use a discussion board instead of this issue board.

**Reply by ZanettaPM** [reply]
Is it not the discussion board ? I thought I choose the correct board for this kind of question.

**Reply by eouia** [reply]
Sorry I was confused due to on mobile

**Reply by ZanettaPM** [reply]
No worries ;)

**Reply by pst-mng** [reply]
I'm just hopping over from Ext2 to Ext3 and I wonder if I really need to have the default calendar in the code but invisible or if it's okay to have it installed.
